subsys/usb/usb_c/usbc_stack.h

```diff
@@ -211,4 +211,37 @@ struct usbc_port_data {
 	void *dpm_data;
 };
 
+#ifdef CONFIG_USBC_CSM_SOURCE_ONLY
+/**
+ * @brief Function that enables the source path either using callback or by the TCPC.
+ * If source and sink paths are controlled by the TCPC, this callback doesn't have to be set.
+ *
+ * @param dev USB-C connector device
+ * @param tcpc Type-C Port Controller device
+ * @param en True to enable the sourcing, false to disable
+ * @return int 0 if success, -ENOSYS if both callback and TCPC function are not implemented.
+ *             In case of error, value from any of the functions is returned
+ */
```

Comment:
It is local function never exposed or used outside, there absolutely no reason to use doxigen comments.

Reply:
You're right, but everything was commented in this file, so I didn't want this one function to be left alone. Do you recommend to remove these comments, so they won't be visible in the generated documentation? (Won't they be included as prototype but undocumented?)

Reply:
I see it is everywhere, it is up to you then. I would just use regular comment style.